Nodejs/Product/Nodejs/NodejsTools.vsct

```diff
@@ -91,6 +91,13 @@
               <CommandFlag>DynamicVisibility</CommandFlag>
           If you do not want an image next to your command, remove the Icon node or set it to <Icon guid="guidOfficeIcon" id="msotcidNoIcon" /> -->
 
+       <Button guid="guidNodeToolsCmdSet" id="cmdidSendFeedback" priority="0x0500" type="Button">
+        <Parent guid="guidSHLMainMenu" id="IDG_VS_HELP_SUPPORT"/>
```

Comment:
This should be in Tools > Node.js Tools - we don't like other ppl cluttering up common menus, so we shouldn't do the same. Additonally, when you do that, you'll also want to add a menu group to separate diagnostic info, survey/news, and send feedback